common/flatpak-run.c

```diff
@@ -1699,6 +1699,11 @@ static const ExportData default_exports[] = {
   {"PERLLIB", NULL},
   {"PERL5LIB", NULL},
   {"XCURSOR_PATH", NULL},
+  {"GST_PLUGIN_PATH_1_0", NULL},
+  {"GST_REGISTRY", NULL},
+  {"GST_REGISTRY_1_0", NULL},
+  {"GST_PLUGIN_PATH", NULL},
+  {"GST_PLUGIN_SYSTEM_PATH", NULL},
```

Comment:
Makes sense to me. But this should probably also cover:
 - GST_PLUGIN_SCANNER / GST_PLUGIN_SCANNER_1_0
 - GST_PLUGIN_SYSTEM_PATH_1_0
 - GST_PRESET_PATH
 - GST_PTP_HELPER / GST_PTP_HELPER_1_0
 - GST_INSTALL_PLUGINS_HELPER